src/irmin-pack/conf.ml

```diff
@@ -33,8 +33,10 @@ module Default = struct
   let index_log_size = 2_500_000
   let readonly = false
   let merge_throttle = `Block_writes
-  let freeze_throttle = `Block_writes
   let indexing_strategy = Indexing_strategy.default
+  let use_fsync = false
```

Comment:
Isn't `use_fsync = true` the sane default? `false` seems risky

Reply:
Why not